### PR TITLE
Upgrade airbrake

### PR DESCRIPTION
**Pull request by kujma10**
INF-708
In a new version of airbrake `developmentEnvironments` field is removed from airbrake object. Instead of that, a new function `addFilter` is introduced.
With help of this function `developmentEnvironments` field is not needed anymore, as we can filter them in `addFilter` function.

### CODE REVIEW

File: test/airbrake_init_test.coffee [hunk 1]
```diff
@@ -51,6 +54,37 @@ describe 'airbrake_init', ->
...
+
+      beforeEach ->
+        configKeys =
+          'projectId': '123'
+          'apiKey': 'myAirbrakeId'
+          'whiteListKeys': ['keys']
+          'developmentEnvironments': ['dev', 'staging']
+          'env': environment()
+        airbrake = AirbrakeInit.initAirbrake(configKeys)
+        sinon.stub(airbrake, '_sendRequest');
```
Comment (willmore) [inline test/airbrake_init_test.coffee]: One should not stub private methods for testing. Especially not libraries which are not under our control.

Comment (sviik) [inline test/airbrake_init_test.coffee]: Actually, it's the 3rd party airbrake which is returned by `AirbrakeInit.initAirbrake()`

Comment (willmore) [inline test/airbrake_init_test.coffee]: Thanks for clarification. That makes me more convinced our method of testing must be changed.

Comment (kujma10) [inline test/airbrake_init_test.coffee]: https://github.com/salemove/node-airbrake-init/pull/3/commits/8a96b4cb9e40f1e00045d9a577858ac26063ee78

File: test/airbrake_init_test.coffee [hunk 2]
```diff
@@ -51,6 +54,48 @@ describe 'airbrake_init', ->
...
+          environment.is -> 'dev'
+
+          it 'returns null', ->
+            notice = errorNotice('an error')
+            expect(filter(notice)).to.be.null
+
+        context 'in production environment', ->
+          environment.is -> 'prod'
+
+          it 'returns notice object with same context', ->
```
Comment (willmore) [inline test/airbrake_init_test.coffee]: 'object' is redundant; `it 'returns notice with same context'` suffices;

File: test/airbrake_init_test.coffee [hunk 3]
```diff
@@ -51,6 +54,48 @@ describe 'airbrake_init', ->
...
+            notice = errorNotice('an error')
+            filteredValue = filter(notice)
```
Comment (willmore) [inline test/airbrake_init_test.coffee]: `Value` suffix seems out of place. What domain type does filter operate on? Use that as suffix instead.

Comment (kujma10) [inline test/airbrake_init_test.coffee]: it's javascript object. we can name it `filteredNotice`

Comment (willmore) [inline test/airbrake_init_test.coffee]: To clear, there are no classes in JS, right? I was looking more for the semantic meaning of the object - which is "notice" I believe. So `filteredNotice` seems appropriate.

File: test/airbrake_init_test.coffee (hunk 2 above)
Comment (kujma10) [inline test/airbrake_init_test.coffee]: https://github.com/salemove/node-airbrake-init/pull/3/commits/87f89f4c77dbda786ad3772ba55e463540d9d4dc

File: test/airbrake_init_test.coffee [hunk 4]
```diff
@@ -51,6 +54,48 @@ describe 'airbrake_init', ->
...
+            expect(filteredValue).not.to.be.null
+            expect(filteredValue.context).not.to.be.null
```
Comment (kujma10) [inline test/airbrake_init_test.coffee]: https://github.com/salemove/node-airbrake-init/pull/3/commits/6e878c78c9a79e275a52f30378ed93eac293815e

File: test/airbrake_init_test.coffee (hunk 3 above)
Comment (kujma10) [inline test/airbrake_init_test.coffee]: https://github.com/salemove/node-airbrake-init/pull/3/commits/ba6316c79054e9c8e4cf2a0e6f70eb869690df70

File: package.json [hunk 5]
```diff
@@ -34,6 +34,7 @@
     "expect": "^1.14.0",
     "memo-is": "0.0.2",
     "mocha": "~> 2.4.5",
+    "should": "^11.1.0",
```
Comment (sviik) [inline package.json]: Do you still need `should`?

Comment (kujma10) [inline package.json]: yes, I couldn't run tests initially without it

**Comment by sviik** [comment]
!squash

**Pull request merged**